Makefile

```
@@ -49,3 +69,15 @@ compress-darwin:
 compress-darwin-arm64:
 	tar -zcvf build/grm-darwin-arm64.tar.gz	-Cbuild/grm-darwin-arm64/	grm
 	rm	-rf	build/grm-darwin-arm64
+
+compress-linux:
+	tar -zcvf build/grm-linux-64.tar.gz	-Cbuild/grm-linux-64/	grm
+	rm	-rf	build/grm-linux-64
+
+compress-linux-32:
+	tar -zcvf build/grm-linux-32.tar.gz	-Cbuild/grm-linux-32/	grm
+	rm	-rf	build/grm-linux-32
+
+compress-linux-arm64:
+	tar -zcvf build/grm-linux-arm64.tar.gz	-Cbuild/grm-linux-arm64/	grm
+	rm	-rf	build/grm-linux-arm64
```

```suggestion
	rm	-rf	build/grm-linux-arm64
	
```
It's better to add black line at the end of the file.